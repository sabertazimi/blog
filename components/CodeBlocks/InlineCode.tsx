import cx from 'classnames'
import styles from './InlineCode.module.css'

interface Props {
  children?: string
  className?: string
}

function InlineCode({ children, className }: Props): JSX.Element {
  return (
    <code
      className={cx(
        className,
        styles.code,
        'dark:text-light dark:shadow-xl dark:shadow-primary',
      )}
    >
      {children}
    </code>
  )
}

export default InlineCode
