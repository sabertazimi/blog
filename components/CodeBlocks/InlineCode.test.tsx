import { render } from '@utils'
import InlineCode from './InlineCode'

test('should render correctly (snapshot)', () => {
  const { container } = render(<InlineCode>Inline Code</InlineCode>)

  expect(container).toMatchSnapshot()
})
