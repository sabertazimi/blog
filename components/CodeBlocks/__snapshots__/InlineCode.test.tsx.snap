// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineCode should render correctly (snapshot) 1`] = `
<div>
  <code
    class="code dark:text-light dark:shadow-xl dark:shadow-primary"
  >
    Inline Code
  </code>
</div>
`;
